xml/nsx-import-third_party.xml

```diff
@@ -263,63 +260,38 @@
-   ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-   &prompt.ardana;cp ipaddress-*.whl
-   ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-   &prompt.ardana;cp warlock-*.whl
-   ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-   &prompt.ardana;cp jsonschema-*.whl
-   ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-   &prompt.ardana;cp functools32-*.whl
-   ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-   &prompt.ardana;cp httplib2-*.whl
-   ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-   &prompt.ardana;cp voluptuous-*.whl
-   ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-   &prompt.ardana;cp tenacity-*.whl
-   ~/third-party/vmware/venvs/vmware-neutronclient/whls/</screen>
+&prompt.ardana;cd /tmp/stage/nsx-whls
+&prompt.ardana;cp vmware_nsx-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp tooz-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp python_openstackclient*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp keystoneauth1-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp python_glanceclient-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp python_cinderclient-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp oslo.utils-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp openstacksdk-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp deprecation-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp jsonpatch-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp jsonpointer-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp python_novaclient-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp pyOpenSSL-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp cryptography-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp cffi-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp pycparser-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp asn1crypto-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp ipaddress-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp warlock-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp jsonschema-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp functools32-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp httplib2-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp voluptuous-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
+&prompt.ardana;cp tenacity-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/</screen>
```

Comment:
I am wondering if there is a good way to make this simpler. My best (not very good) idea was this:

```
&prompt.ardana; cp {vmware_nsx-,tooz-,python_openstackclient,keystoneauth1-,\
python_glanceclient-,[...omitted files...]}*.whl \
  ~/third-party/vmware/venvs/vmware-neutronclient/whls/
```

But the current version is so repetitive!